ext/oci8/oci8_statement.c

```diff
@@ -42,6 +42,9 @@ typedef ub8 oci_phpsized_int;
 typedef ub4 oci_phpsized_int;
 #endif
 
+#define Z_OCILOB_DESCRIPTOR_P(zv) OBJ_PROP_NUM(Z_OBJ_P(zv), 0)
```

Comment:
IMO these macros should be added to the php_oci8.h header instead if they has to be shared internally.